### ISSUE TITLE
color contrast ratio

### ISSUE THREAD

**Issue by Onyekaokafor** [JordisGithub/desn#73]
homepage/landing page, Upcoming events time schedule 10am-2pm text and background color does not meet the required color contrast ratio. While the first image meet the required contract ratio. I suggest we use the blue and white HEX value for background and text.
